eq-author-api/config/knexfile.js

```diff
@@ -1,35 +1,7 @@
 module.exports = {
-  development: {
-    client: "postgresql",
+  client: "postgresql",
+  connection: {
     connection: process.env.DB_CONNECTION_URI
   },
-
-  test: {
-    client: "postgresql",
-    connection: process.env.DB_CONNECTION_URI
-  },
-
-  staging: {
-    client: "postgresql",
-    connection: process.env.DB_CONNECTION_URI,
-    pool: {
-      min: 2,
-      max: 10
-    },
-    migrations: {
-      tableName: "knex_migrations"
-    }
-  },
-
-  production: {
-    client: "postgresql",
-    connection: process.env.DB_CONNECTION_URI,
-    pool: {
-      min: 2,
-      max: 10
-    },
-    migrations: {
-      tableName: "knex_migrations"
-    }
-  }
+  acquireConnectionTimeout: 5000
```

Comment:
Why do we need this?

Reply:
Default connection timeout is 60000 which I thought was quite excessive. if we haven't go a connection in 5 seconds we arn't getting one